dask/dataframe/dispatch.py

```diff
@@ -140,3 +141,19 @@ def make_meta(x, index=None, parent_meta=None):
 def union_categoricals(to_union, sort_categories=False, ignore_order=False):
     func = union_categoricals_dispatch.dispatch(type(to_union[0]))
     return func(to_union, sort_categories=sort_categories, ignore_order=ignore_order)
+
+
+class DataFrameBackendDispatch(BackendDispatch):
+    def get_backend(self):
+        return config.get("dataframe.backend.library") or "pandas"
+
+    @property
+    def allow_fallback(self):
+        return config.get("dataframe.backend.allow-fallback", True)
```

Comment:
I believe this config should always be set, so we don't need the default value 
```suggestion
        return config.get("dataframe.backend.allow-fallback")
```